MicrosoftFluentUI.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'MicrosoftFluentUI'
-  s.version          = '0.1.28'
+  s.version          = '0.1.29'
```

Comment:
We have to update the source (line 8) too:

s.source       = { :git => "https://github.com/microsoft/fluentui-apple.git", :branch => 'main_0.1', :tag => "#{s.version}_main_0.1" }

Reply:
for my knowledge, can the branch and tag work together in podspec? i thought it was oneway or the other

Reply:
It can. We just need to make sure the tag is created on the main_0.1 branch.
I suggested the suffix 0.1.29**_main_0.1** just to match the branch name so we don't get confused in the future.

Reply:
> They can. We just need to make sure the tag is created on the main_0.1 branch.
> I suggested the suffix 0.1.29**_main_0.1** just to match the branch name so we don't get confused in the future.